src/subclass/input_stream.rs

```diff
@@ -295,6 +301,58 @@ mod tests {
         }
     }
 
+    impl SeekableImpl for SimpleInputStream {
+        fn tell(&self, _seekable: &Seekable) -> i64 {
+            *self.pos.borrow() as i64
+        }
+
+        fn can_seek(&self, _seekable: &Seekable) -> bool {
+            true
```

Comment:
Isn't there supposed to be a limit to how much you can seek? Or is it just a global "is this object seekable"?

Reply:
Just globally at this very moment